content/pt-br/community/static/cncf-code-of-conduct.md

```diff
@@ -13,32 +14,62 @@ todos, independentemente do nível de experiência, gênero, identidade e expres
 orientação sexual, deficiência, aparência pessoal, tamanho do corpo, raça, etnia, idade,
 religião ou nacionalidade.
 
-Exemplos de comportamento inaceitável pelos participantes incluem:
+## Escopo 
 
-* O uso de linguagem ou imagens sexualizadas
-* Ataques pessoais
-* Trolling ou insultar / comentários depreciativos
+Este código de conduta aplica-se tanto nos espaços do projeto como nos espaços públicos
+quando um indivíduo está representando o projeto ou sua comunidade.
+
+### Eventos CNCF
+
+Eventos CNCF, ou eventos organizados pelo Linux Foundation com equipe profissional de eventos, são regidos pelo Linux Foundation [Código de Conduta] (https://events.linuxfoundation.org/code-of-conduct/) disponível na página do evento. Isso foi projetado para ser utilizado em conjunto com o Código de Conduta do CNCF.
```

Comment:
Some suggestions:
* "Eventos da CNCF" instead of "Eventos CNCF".
* "são regidos pelo Código de Conduta da Linux Foundation".
* Perhaps use "pela Linux Foundation" too since it is a "fundação" (feminine noun), event if not translated.

One problem:
* There's an extra space between the square and the round brackets in the link that is currently breaking rendering.